libs/@blockprotocol/type-system/rust/src/schema/entity_type/raw.rs

```diff
@@ -66,7 +67,7 @@ pub struct EntityType<'a> {
     #[serde(with = "links", default, skip_serializing_if = "HashMap::is_empty")]
     links: Links,
     #[serde(default, skip_serializing_if = "<[_]>::is_empty")]
-    examples: Cow<'a, [HashMap<String, serde_json::Value>]>,
+    examples: Cow<'a, [HashMap<BaseUrl, JsonValue>]>,
```

Comment:
We were going to remove `examples`, right? Is there a ticket for that?

Reply:
Yes: BP-63
I wanted to remove it as part of this PR but some more parts had to be adjusted for that. Also, I need to check if the production DB is using `examples` anywhere. I'm going to do that in a follow-up.